rn-cli.config.js

```diff
@@ -0,0 +1,9 @@
+// workaround for react-native-vector-icons
```

Comment:
could you elaborate why we need this?

Reply:
Apparently some changes in Metro's file resolving behavior surfaced packaging issues in react-native-vector-icons. This file works around this by ignoring the offending files.

A longer explanation [here](https://github.com/facebook/metro/issues/139#issuecomment-366213751).

Reply:
I understand the files ignored are test fixtures, so there should be no harm in ignoring them in our builds.